geo-types/src/line_string.rs

```diff
@@ -281,6 +288,61 @@ impl<T: CoordinateType> IndexMut<usize> for LineString<T> {
     }
 }
 
+#[cfg(test)]
+impl<T> RelativeEq for LineString<T>
+where
+    T: CoordinateType + Float + AbsDiffEq,
+{
+    // type Epsilon = T;
```

Comment:
Does this comment mean something or just leftover and can can be removed?

Reply:
Thanks .. I will fix this